src/libraries/System.Private.Xml/src/System/Xml/Schema/FacetChecker.cs

```diff
@@ -1477,7 +1466,7 @@ private bool MatchEnumeration(string value, ArrayList enumeration, XmlSchemaData
                     {
                         return new XmlSchemaException(SR.Sch_EmptyAttributeValue, string.Empty);
                     }
-                    if (!LanguagePattern.IsMatch(s))
+                    if (!LanguageRegex().IsMatch(s))
```

Comment:
(Sorry if it is not obvious to me)
Is it equivalent to the cached instance as before, or will it create a new one for subsequent executions (and we don't care here because it is not considered as a hot path)?

Reply:
Perfect, thank you!